src/components/Header/EmcLogo.astro

```diff
@@ -0,0 +1,8 @@
+---
+const { width } = Astro.props;
+---
+
+<svg width={width} viewBox="0 0 999 281">
```

Comment:
Add alt text
  
  ```
  <svg role="img">
      <title>Email Markup Consortium</title>
 ```

Reply:
Looks like I didn't hit send on that second comment before submitting the review

Reply:
Good call. Added.

Reply:
Sorry I wasn't very clear in the first message, i would also want a `role="img"` on the `<svg>` tag